### PR TITLE
LPS-196940 The hide icon (eye) is being displayed despite being hovered or not

### PR DESCRIPTION
**Pull request by Artartur**
Related Issue: [LPS-196940](https://liferay.atlassian.net/browse/LPS-196940)

### CODE REVIEW

**Comment by naraandrade** [comment]
Just started reviewing :)

**Comment by Artartur** [comment]
ci:test:sf

File: modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ModelBuilder/LeftSidebar/LeftSidebar.scss [hunk 1]
```diff
@@ -54,6 +53,18 @@
 		padding: 0 !important;
 	}
 
+	.lfr-objects__model-builder-left-sidebar-show-folders-button {
```
[the remark this answers is not in the document]
Comment (Artartur) [reply modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ModelBuilder/LeftSidebar/LeftSidebar.scss]: Done!

File: modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ModelBuilder/LeftSidebar/LeftSidebar.scss [hunk 2]
```diff
@@ -5,7 +5,6 @@
 	flex-direction: column;
 	padding: 1rem;
 	position: inherit;
-
```
[the remark this answers is not in the document]
Comment (Artartur) [reply modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ModelBuilder/LeftSidebar/LeftSidebar.scss]: Yeah, my bad, I've added it again

**Comment by Artartur** [comment]
ci:test:sf

**Comment by naraandrade** [comment]
Just started reviewing :)

**Comment by naraandrade** [comment]
ci:forward:force

**Comment by naraandrade** [comment]
ci:forward:force

**Pull request closed without merging**